src/mapsettings.h

```diff
@@ -75,6 +75,11 @@ class MapSettings : public QObject
       */
     Q_PROPERTY( QgsCoordinateReferenceSystem destinationCrs READ destinationCrs WRITE setDestinationCrs NOTIFY destinationCrsChanged )
 
+    /**
+      * The string of the QgsUnitTypes::DistanceUnits CRS of destination coordinate reference system.
+      */
+    Q_PROPERTY( QString abbreviatedStringOfDistanceUnit READ abbreviatedStringOfDistanceUnit )
```

Comment:
Will that also need a changed signal emitted when the mapsettings.destinationcrs changes?

Reply:
I don't think so. This is emitted with the data of the current destinationcrs...

Reply:
But possibly you think about a use case I don't considered.

Reply:
Ah your are right. I haven't considered that be don't emit the abbreviatedString with the distanceCrs anymore like before...